README.rst

```diff
@@ -51,9 +53,10 @@ methods.
 Installation (WIP)
 ------------------
 
-Once the first release is done, skrub can be easily installed via `pip`::
+There are currently no PiPy releases.
+You can still install the package from the GitHub repository with:
 
-    pip install skrub
+    pip install git+https://github.com/skrub-data/skrub.git
```

Comment:
After an IRL discussion with @GaelVaroquaux, our strategy is to be tested and used only by people with a significant technical and statistical grasp during the early days of skrub. These people will know how to install and test the project without us helping them. We can discuss that further.

Reply:
I guess we revert this when skrub will be available in pypi